example_analysis/2.configure_sbs_params.ipynb

```diff
@@ -62,20 +62,20 @@
     "    image_segmentation_annotations,\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    from lib.sbs_process.align_cycles import align_cycles
</pre><p>Don't change back to old sbs_process name!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheeseman-lab/brieflow/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>